src/components/newsletter/newsletter-sign-up.test.tsx

```diff
@@ -0,0 +1,161 @@
+import React from "react";
+import { create } from "react-test-renderer";
+import NewsletterSignUp from "./newsletter-sign-up";
+import "@testing-library/jest-dom";
+import addToMailchimp from "gatsby-plugin-mailchimp";
+import { render, act, fireEvent, waitFor } from "@testing-library/react";
+
+jest.mock("gatsby-plugin-mailchimp");
+
+describe("NewsletterSignUp", () => {
+  it("renders correctly", () => {
+    const component = create(<NewsletterSignUp />).toJSON();
+    expect(component).toMatchSnapshot();
+  });
+
+  it("Name and email errors are not visible on render", async () => {
+    const { getByText } = render(<NewsletterSignUp />);
+
+    expect(getByText("Please enter your name.")).not.toBeVisible();
+    expect(getByText("Please enter your email.")).not.toBeVisible();
+  });
+
+  it("Displays error name field is submitted blank", async () => {
```

Comment:
think these need a `"when"` in the name to read well
`it("displays email error message when blank email is submitted", …`

Reply:
yeah this was a terrible copy/paste job all over 🙈